apps/i18n/common/en_us.json

```diff
@@ -19,6 +19,7 @@
   "clearPuzzleConfirmHeader": "Are you sure you want to start over?",
   "codeMode": "Code",
   "codeTooltip": "See generated JavaScript code.",
+  "completedWithoutRecommendedBlock": "Congratulations! You completed Puzzle {puzzleNumber}. (But you could use a different block for stronger code.)",
```

Comment:
"stronger code"? Also, this string is different than what i see in the screenshots